extras/TestCohortDiagnostics.R

```diff
@@ -0,0 +1,110 @@
+library(magrittr)
+# Set up
+baseUrl <- Sys.getenv("BaseUrl")
+cohortIds <- c(18345,18346,14906,18351,18347,18348,14907,
+               18349,18350,18352,17493,17492,14909,18342,
+               17693,17692,17695,17694,17720, 
+               21402)
+
+# get specifications for the cohortIds above
+webApiCohorts <-
+        ROhdsiWebApi::getCohortDefinitionsMetaData(baseUrl = baseUrl) %>%
+        dplyr::filter(.data$id %in% cohortIds)
+
+# compile them into a data table
+studyCohorts <- list()
+for (i in (1:nrow(webApiCohorts))) {
+        cohortId <- webApiCohorts$id[[i]]
+        cohortDefinition <-
+                ROhdsiWebApi::getCohortDefinition(cohortId = cohortId, baseUrl = baseUrl)
+        df <- tidyr::tibble(
+                id = cohortId,
+                createdDate = webApiCohorts$createdDate[[i]],
+                modifiedDate = webApiCohorts$modifiedDate[[i]],
+                description = webApiCohorts$description[[i]],
+                name = stringr::str_trim(stringr::str_squish(cohortDefinition$name)),
+                expression = cohortDefinition$expression %>% 
+                        RJSONIO::toJSON(digits = 23),
```

Comment:
I think you can skip the "toJSON" conversion for the expression here as it has the JSON you need, correct?